app/views/hyrax/una_chapters_and_book_sections/_una_chapters_and_book_section.html.erb

```diff
@@ -0,0 +1,2 @@
+<%# This is a search result view %>
+<%= render 'catalog/document', document: una_chapters_and_book_section, document_counter: una_chapters_and_book_section_counter %>
```

Comment:
Can you switch the single quotes for these partials.

Reply:
I genuinely don't like you sometime you know <3